Library/Formula/adwaita-icon-theme.rb

```diff
@@ -1,17 +1,7 @@
-require "formula"
-
-class GnomeIconTheme < Formula
+class AdwaitaIconTheme < Formula
```

Comment:
We can't rename formulae as people won't be upgraded. Instead let's just add an alias to the new name.